web/src/engine/websites/UnionMangas.ts

```diff
@@ -3,25 +3,31 @@ import icon from './UnionMangas.webp';
 import { type Chapter, DecoratableMangaScraper, type Manga } from '../providers/MangaPlugin';
 import * as Common from './decorators/Common';
 import * as MangaStream from './decorators/WordPressMangaStream';
+import { Fetch, FetchRequest } from '../FetchProvider';
 
-@Common.MangaCSS(/^https?:\/\/guimah\.com\/manga\//, 'div.manga-pagina div.row div.col-md-12 h2')
-@Common.MangasMultiPageCSS('/lista-mangas/visualizacoes/{page}', 'div.tamanho-bloco-perfil div.lista-mangas > a:last-of-type')
+@Common.MangaCSS(/^https?:\/\/unionmangasbr\.top\/manga\//, 'div.manga-perfil-novo div.row div.col-md-12 h2')
+@Common.MangasMultiPageCSS('/lista-mangas/visualizacoes/{page}', 'div.tamanho-bloco-perfil div.lista-itens > a:last-of-type')
 @MangaStream.PagesSinglePageCSS([/banner_/], 'img.img-manga')
 @Common.ImageAjax()
 
 export default class extends DecoratableMangaScraper {
 
     public constructor() {
-        super('unionmangas', `UnionMangas`, 'https://guimah.com', Tags.Language.Portuguese, Tags.Media.Manga, Tags.Source.Aggregator);
+        super('unionmangas', `UnionMangas`, 'https://unionmangasbr.top', Tags.Language.Portuguese, Tags.Media.Manga, Tags.Source.Aggregator);
     }
 
     public override get Icon() {
         return icon;
     }
 
+    //override Initialize because website is so damn slow, probably because ads and tracking
+    public override async Initialize(): Promise<void> {
+        await Fetch(new FetchRequest(this.URI.href));
```

Comment:
It seems this `fetch` does not have any use, so it should be removed, just leave the function body empty